webapp/src/components/call_widget/component.tsx

```diff
@@ -1382,15 +1380,16 @@ export default class CallWidget extends React.PureComponent<Props, State> {
                 }}
                 ref={this.node}
             >
-                <div style={this.style.status as CSSProperties}>
-                    <div style={{position: 'absolute', bottom: 'calc(100% + 4px)', width: '100%', transform: 'translateZ(-10px)'}}>
```

Comment:
I think all you need to do is remove the transform, that should prevent the div from being unclickable -- not stacking context, per se... Kind of curious, why were you using it?

Reply:
The transform was added as a workaround to make the notification appear from behind the main widget body. I was mostly following https://fjolt.com/article/css-make-child-appear-behind-parent

What am I missing?

Reply:
Oh, I didn't realize that was for notifications. I'm not sure what you're missing ( :) ) but I was missing that info. But I guess the transform causes a stacking context, I'm not sure why that alone would cause this (and in safari alone).

Reply:
The stacking context was already there since we set `z-index` on the root element (`#calls-widget`) to make it show in front of anything else in Mattermost. And the notifications were always showing above the widget until I implemented non-blocking errors and refined the animation which looks much nicer if they come from below. At that point I couldn't find a proper way to make the child element display above its parent so resorted to the transform hack which doesn't work in Safari for some reason (maybe https://stackoverflow.com/questions/25055505/css-transition-transform-z-index-conflict-in-safari-works-on-chrome-ff). Either way, now it's much better as it only relies on z-index.

Reply:
That is pretty cool -- good work here, thanks!